openshift/ansible/__init__.py

```diff
@@ -35,15 +40,25 @@ def __str__(self):
 
 
 class OpenShiftAnsibleModule(AnsibleModule):
+    """
+    Abstraction of an OpenShift Ansible Module.
+    """
+
     def __init__(self, openshift_type):
+        """
```

Comment:
I have some snippets in my editor which I use most the time but nothing fancy. What editor do you use?